PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -2500,8 +2515,8 @@ void FindReplaceDlg::doDialog(DIALOG_TYPE whichType, bool isRTL, bool toShow)
 {
 	if (!isCreated())
 	{
-		create(IDD_FIND_REPLACE_DLG, isRTL);
 		_isRTL = isRTL;
+		create(IDD_FIND_REPLACE_DLG, isRTL);

```

Comment:
This order is MUST. Member _isRTL must be set in order to use at line 690, 694, 698. Otherwise, below statement has no effect.
p = getTopPoint(::GetDlgItem(_hSelf, IDCANCEL), !_isRTL)); which will not solve the RTL issue.